phantomJS.py

```diff
@@ -0,0 +1,102 @@
+import sys, re, time
+from bs4 import BeautifulSoup
+from urlparse import urlparse
+from selenium import webdriver
+
+visitedDomains = set()
+foundEmails = set()
+clickedLinks = set()
+originDomain = ""
+
+def find_email_address2(domain, visitedPages):
+    visitedPages -= 1
+    domain = 'http://'+domain
+    if domain in visitedDomains:
+        return
+    o = urlparse(domain)
+    #verify we stay in domains that match the input netlocation
+    if originDomain not in o.netloc:
+        return
+    print("visiting: "+domain)
+    driver = webdriver.PhantomJS()
+    driver.set_window_size(1124, 850)
+    driver.get(domain)
+    page_source = driver.page_source
+    soup = BeautifulSoup(page_source, "lxml")
+    #find email addresses on page
+    domains = check_page(soup)
+    visitedDomains.add(domain)
+    for i in set(domains):
+        foundEmails.add(i)
+    #find links in page
+    links = find_links(soup, domain)
+
+    #click on link
+    spans = driver.execute_script("return document.querySelectorAll('span')");
```

Comment:
An ng-click can exist on elements other than a span, and an ng-click doesn't necessarily mean a route change.  This means you'd queue up "links" that weren't really links.  In the relatively simple case of the jana.com, this probably works fine, though :smile: